conf/zeppelin-site.xml.template

```diff
@@ -184,7 +184,7 @@
 
 <property>
   <name>zeppelin.interpreter.group.order</name>
-  <value>"spark,md,angular,sh,livy,alluxio,file,psql,flink,python,ignite,lens,cassandra,geode,kylin,elasticsearch,scalding,jdbc,hbase</value>
+  <value>spark,md,angular,sh,livy,alluxio,file,psql,flink,python,ignite,lens,cassandra,geode,kylin,elasticsearch,scalding,jdbc,hbase</value>
```

Comment:
@zjffdu Actually it's not a scope of this change, I just noticed `bigquery` is not included in here. Can we add `bigquery` using this PR?